it-follows/audio.lua

```diff
@@ -0,0 +1,28 @@
+--- Audio Module
+--- Handles BGM, sound effects and volume parametes
+local Audio = {}
+Audio.__index = Audio
+
+function Audio:new()
+  local instance = setmetatable({}, Audio)
+  local success, bgm = pcall(love.audio.newSource, 'assets/audio/bgm.mp3', 'stream')
+    if not success then
+        print("Erro ao carregar bgm: " .. tostring(bgm))
+    end
+        instance.bgm = bgm
+  -- TODO: add sound effect sources
+  bgm:play()
+  return instance
+end
+
+function Audio:bgmUpdate(status)
+  if status == "started" then
+    self.bgm:play()
+    return
+  end
+  self.bgm:stop()
+  -- TODO: handle change bgm at certain score milestones
+  -- TODO: compose original music =D
+end
```

Comment:
`bgmUpdate` is not following naming pattern, and also depends on status, a piece of data that is outside of its context.

What I would do:
1. Create a `Audio.status -- options 'playing' | 'stopped'`
2. Remove this function and create `play/stop` functions for this module
3. Add logic to play and stop audio on the `game.lua` file directly

We can do a pair-programming session if you like